src/services/participantsService.js

```diff
@@ -0,0 +1,52 @@
+import api from "../api/api";
+import store from "../store/store";
+import { setUsers } from "../store/Participants";
+
+class ParticipantsService {
+  async syncDataFromChats(chatsIds) {
```

Comment:
to be honest, I think it will be easier to sync participants for chats that we obtain in  `ConversationsService `.
That will be more durable and clean.

Because I think having `if (!isDeepEqual(currentValue, previousValue)) {` may not work properly. 

So, probably `ParticipantsService ` is not required at this stage and we simply can sync participants in `ConversationsService`